src/reactions.jl

```diff
@@ -163,7 +163,7 @@ Get rate constant of mass action kineticLaws
 function get_massaction(kl::Num, reactants::Union{Vector{Num}, Nothing},
         stoich::Union{Vector{<:Real}, Nothing})
     function check_args(x::SymbolicUtils.BasicSymbolic{Real})
-        check_args(Val(SymbolicUtils.istree(x)), x)
+        check_args(Val(SymbolicUtils.TermInterface.iscall(x)), x)
```

Comment:
Also, do you know why this does not fix [this](https://github.com/SciML/SBMLToolkit.jl/actions/runs/9681043717/job/26710697796?pr=159#step:6:1200), but instead causes failure in Downgrade.yaml? And why does Downgrade `julia-actions/julia-runtest@v1`, if it is already present in CI.yaml?

Reply:
There might be more in ModelingToolkit to handle.